src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,43 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.15.0
+
+### New features
+
+* Submariner can now be installed on a dual-stack Kubernetes cluster. Currently, it does not support IPv6, but connectivity and
+  service-discovery would continue to work normally using IPv4 addresses.
+* Added a `subctl recover-broker-info` command to recover lost `broker-info.subm` file.
+* It is now possible to customize the default TCP MSS clamping value set by Submariner also for non-Globalnet deployments.
+* The `subctl gather` command will now gather iptables logs for Calico and kindnet CNIs.
+* The `subctl gather` command now collects the `ipset` info from all the cluster nodes.
+* The `subctl diagnose` command now validates the Calico IPPool configuration to match with Submariner requirements.
+* The following deprecated commands and variants have been removed:
+  * `subctl benchmark`’s `--kubecontexts` option (use `--context` and `--tocontext` instead)
+  * `subctl benchmark`’s `--intra-cluster` option (specify a single context to run intra-cluster benchmarks)
+  * `subctl benchmark` with two `kubeconfigs` as command-line arguments
+  * `subctl cloud`’s `--metrics-ports` option
+  * `subctl deploy-broker`’s `--broker-namespace` option (use `--namespace` instead)
+  * `subctl diagnose firewall metrics` (this is checked during deployment)
+  * `subctl diagnose firewall intra-cluster` with two `kubeconfigs` as command-line arguments
+  * `subctl diagnose firewall inter-cluster` with two `kubeconfigs` as command-line arguments
+  * `subctl gather`’s `--kubecontexts` option (use `--contexts` instead)
+
+### Other changes
+
+* The namespace is now included in `EndpointSlice` names to avoid conflicts between services with the same name in multiple namespaces.
+* Fix issues in Azure cloud prepare:
+  * Machine set names are now based on region + uuid and limited to 20 characters to prevent issues with long cluster names.
+  * Machine set creation and deletion logic was updated to prevent creation of multiple gateway nodes.
+  * Image names are now retrieved from existing machine sets.
+* Fix stale IPtable rules along with globalIP leak which can sometimes happen when a GlobalEgressIP is created and immediately deleted as
```

Comment:
On L16 above we refer to "iptables" but here "IPtable". We should be consistent.

```suggestion
* Fixed stale IPtable rules and a global IP leak which can sometimes happen when a `GlobalEgressIP` is created and immediately deleted as
```